File: go_fizzbuzz/fizzbuzz.go
package fizzbuzz

import (
  "fmt"
)

// Calculates multiples of 3, and 5, then prints Fizz on multiples 3 and prints
// Buzz on multiples of 5. Then on multiples of 3 and 5, print FizzBuzz.
func FizzBuzz(fizzBuzz int) (output string) {
  if (fizzBuzz % 3 == 0 && fizzBuzz % 5 == 0 && fizzBuzz != 0) {
    output = "FizzBuzz"
  } else if (fizzBuzz % 3 == 0 && fizzBuzz != 0) {
    output = "Fizz"
  } else if (fizzBuzz % 5 == 0 && fizzBuzz != 0) {
    output = "Buzz"
  } else {
    output = fmt.Sprintf("%v", fizzBuzz)
  }
  return output
}

func UseFizzBuzz() {
  fizzBuzz := 15
  for num := 0; num <= fizzBuzz; num++ {
    fmt.Println(FizzBuzz(num))
  }
}

File: python_fizzbuzz/fizzbuzz/fizzbuzz.py
#!/usr/bin/env
#
#
#

def fizzbuzz( num = 100 ):
    '''
    Calculate the modulus of 3's and 5's printing Fizz on 3's and Buzz on 5's
    then printing FizzBuzz on modulus of 3 and 5.
    Returns the output of either Fizz, Buzz, or FizzBuzz. 
    '''
    if num % 3 == 0 and num != 0 :
        output = "Fizz"
    elif num % 5 == 0 and num != 0 :
        output = "Buzz"
    elif num % 3 == 0 and num % 5 == 0 and num != 0 :
        output = "FizzBuzz"
    else:
        output = "%s", (num)
    # end of if...elif...end
    return output
# end of fizzbuzz